README.adoc

```diff
@@ -31,16 +31,13 @@ This is the server component of the AeroGear Mobile Security Service. It is a RE
 
 == Prerequisites
 
-.Prerequisites
 |===
 |https://golang.org/doc/install[Install Golang]
 |https://github.com/golang/go/wiki/SettingGOPATH[Ensure the $GOPATH environment variable is set]
-|https://docs.docker.com/compose/install/[Install the dep package manager]
+|https://golang.github.io/dep/docs/installation.html/[Install the dep package manager]
 |https://docs.docker.com/compose/install/[Install Docker and Docker Compose]
 |===
 
-See the https://aerogear.org/community/#guides[Contributing Guide] for more information.
-
 == Getting Started
 
 If you'd like to simply run the entire application in `docker-compose`, follow link:#Run-Entire-Application-with-Docker-Compose[these instructions].
```

Comment:
@camilamacedo86 apologies for pinging on changes that weren't part of your pr but this link here is not working correctly. Would you mind updating it here?

Reply:
Hi @laurafitzgerald I tested it now and it is working perfectly. 
See here: https://github.com/aerogear/mobile-security-service/blob/1d30f18d17b6525979826628045290a8ca3870cf/README.adoc

Reply:
@camilamacedo86 I'm sure why but when I click on the `these instructions` link the url updates but it doesn't navigate to the section.

Reply:
<img width="1096" alt="screenshot 2019-02-25 at 10 47 08" src="https://user-images.githubusercontent.com/7708031/53332246-bd3e7300-38ea-11e9-9552-5420b7b88480.png">

Reply:
Fixed 👍

Reply:
Confirmed the link works correctly now. Thanks.